core/src/main/java/org/apache/accumulo/core/file/blockfile/cache/lru/LruBlockCacheConfiguration.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.accumulo.core.file.blockfile.cache.lru;
+
+import java.util.Map;
+
+import org.apache.accumulo.core.conf.AccumuloConfiguration;
+import org.apache.accumulo.core.conf.Property;
+import org.apache.accumulo.core.file.blockfile.cache.BlockCacheConfiguration;
+
+public final class LruBlockCacheConfiguration extends BlockCacheConfiguration {
+
+  /** Default Configuration Parameters */
+
+  /** Backing Concurrent Map Configuration */
+  public static final Float DEFAULT_LOAD_FACTOR = 0.75f;
+  public static final Integer DEFAULT_CONCURRENCY_LEVEL = 16;
+
+  /** Eviction thresholds */
+  public static final Float DEFAULT_MIN_FACTOR = 0.75f;
+  public static final Float DEFAULT_ACCEPTABLE_FACTOR = 0.85f;
+
+  /** Priority buckets */
+  public static final Float DEFAULT_SINGLE_FACTOR = 0.25f;
+  public static final Float DEFAULT_MULTI_FACTOR = 0.50f;
+  public static final Float DEFAULT_MEMORY_FACTOR = 0.25f;
+
+  // property names
+  private static final String PREFIX = Property.GENERAL_ARBITRARY_PROP_PREFIX + "cache.block.lru.";
```

Comment:
I think it would be nice if the super class handled things in such a way that all subclasses did not have deal with the full property name.  Not sure the best way to do this.  One way I thought of this following.

In `LruBlockCacheConfiguration` constuctor call super constructor with a desired prefix.  This prefix will be used with all gets.
```java
public LruBlockCacheConfiguration(CacheType ct, AccumuloConfiguration conf) {
    super(ct, "lru", conf);
    // this will actually get "general.custom.cache.lru.all.acceptable.factor" ... saving the subclass some work... also encourages use of a standard prefix
    this.acceptableFactor = getOrDefault("acceptable.factor", DEFAULT_ACCEPTABLE_FACTOR);

   // could suppor the CacheType concept... this could get 
   // "general.custom.cache.lru.index.acceptable.factor" or "general.custom.cache.lru.data.acceptable.factor"
   // depending on value of ct
    this.acceptableFactor = getOrDefault(ct, "acceptable.factor", DEFAULT_ACCEPTABLE_FACTOR);
```